### ISSUE TITLE
Songs: Invisible song lists on song entries

### ISSUE THREAD

**Issue by andreoda** [VocaDB/vocadb#1611]
Some song entries have song lists that are not possible to see when clicking on the text.

For example: https://vocadb.net/S/198721
It shows the number "2" but when you click on it, the pop up shows no lists.

I am not sure if this is related to the privacy setting to hide song and album ratings or if it is another bug (maybe it saves already deleted lists?).
In any case, it is probably better to **not** show the "Pools and song lists", if they are invisible to see.

**Comment by FinnRG** [comment]
@andreoda  How about adding a text like "(2 lists hidden)"? This should satisfy both your and @CatgirlFrostmoon's request.

**Comment by andreoda** [comment]
Yes, that would make it not look like a bug anymore, since it is odd to see the empty box.

**Comment by CatgirlFrostmoon** [comment]
I'd be down for that, too! A little indication that *something *is there is
all we need. :3
